.github/workflows/ci-build.yml

```diff
@@ -52,8 +58,14 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           SUPPORT_DIR: ${{ github.workspace }}/target/support
+      - name: Publish Test Report for ${{ matrix.docker_db_version }}
+        uses: scacap/action-surefire-report@v1
+        if: ${{ always() }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I this the one that adds Github actions annotations below for failed tests? That looks great!

Reply:
Yes, exactly :)